hadoop-hdds/common/src/main/resources/ozone-default.xml

```diff
@@ -347,6 +347,14 @@
         Default is 5s.
     </description>
   </property>
+  <property>
+    <name>hdds.ratis.leader.first.election.minimum.timeout.duration</name>
```

Comment:
`hdds.ratis.leader.first-election.minimum.timeout.duration`

to be more like the one for SCM